src/drt/src/dr/FlexGridGraph.h

```diff
@@ -1113,9 +1114,14 @@ class FlexGridGraph
 
   // locations of access points. The vector is indexed by layer number.
   frVector<std::set<Point>> ap_locs_;
+  std::ofstream dump_file_;
+  bool debug_{false};
```

Comment:
How is debug_ enabled?

Reply:
not a user option. It will be changed internally in the code based on the criteria that the developer decides on. just like debugMazeIter.